components/layout_2020/positioned.rs

```diff
@@ -21,9 +21,15 @@ pub(crate) struct AbsolutelyPositionedBox {
     pub contents: IndependentFormattingContext,
 }
 
+#[derive(Default)]
+pub(crate) struct PositioningContext<'box_tree> {
+    /// With `position: absolute`
+    pub abspos: Vec<CollectedAbsolutelyPositionedBox<'box_tree>>,
+}
+
 #[derive(Debug)]
-pub(crate) struct AbsolutelyPositionedFragment<'box_> {
-    absolutely_positioned_box: &'box_ AbsolutelyPositionedBox,
+pub(crate) struct CollectedAbsolutelyPositionedBox<'box_tree> {
```

Comment:
What about `HoistedAbsolutelyPositionedBox`? These exist only when being hoisted up the tree to reach their final absolute containing block, right?